packages/server/modules/core/repositories/objects.ts

```diff
@@ -53,6 +55,24 @@ export const getObjectFactory =
       .first()
   }
 
+export const getFormattedObjectFactory =
+  (deps: { db: Knex }): GetFormattedObject =>
+  async ({ streamId, objectId }) => {
+    const res = await tables
+      .objects(deps.db)
+      .where({ streamId, id: objectId })
+      .select('*')
+      .first()
+    if (!res) return null
+
+    // TODO: Why tho? A lot if not most of places already just use getObjectFactory,
```

Comment:
@didimitrie @gjedlicska Any comments on this one? There's only like 2-3 places that use this older getObject fn, most use getObjectFactory which doesn't do any of this extra formatting and apparently that's fine enough?

Reply:
the `/objects/:streamId/:objectId/single` endpoint is still actively used, lets not change behavior for it